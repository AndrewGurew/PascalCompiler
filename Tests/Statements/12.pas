begin
for i:=1 to n do
a:=3;
end