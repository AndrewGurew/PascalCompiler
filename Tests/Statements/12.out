Unknown identifier i
Unknown identifier i
Unknown identifier n
Unknown identifier a
Unknown identifier a
Main block
 ⎬ For satement
  ⎬i
  ⎬1
  ⎬n
  ⎬ Assign
   ⎬a
   ⎬3
End of Main block
